wit/blobstore.wit

```diff
@@ -1,27 +1,86 @@
 // wasi-cloud Blobstore service definition
 interface blobstore {
-  use container.{container};
-  use types.{error, container-name, object-id};
+  use wasi:io/streams@0.2.0.{
+    input-stream,
+    output-stream,
+  };
+
+  use types.{
+    container-metadata,
+    incoming-value,
+    object-metadata,
+    outgoing-value,
+  };
 
   // creates a new empty container
-  create-container: func(name: container-name) -> result<container, error>;
+  create-container: func(name: string) -> result<container, string>;
```

Comment:
perhaps we want an enum type for the error of blob store operations?

Reply:
I thought about that, but didn't want to do _too_ many changes here. I can open an issue and address in a follow up along with #5